test/test_overrides.py

```diff
@@ -472,13 +472,37 @@ def test_generator(func, override):
             func_args += [TensorLike(), TensorLike()]
 
         def test(self):
-            self.assertEqual(func(*func_args), -1)
-
+            try:
+                self.assertEqual(func(*func_args), -1)
+            except TypeError as e:
```

Comment:
Could you add a comment here on why this can fail / is expected to fail? And also how people should go about adding things to this list if (e.g.) error messages change or a new C extension appears?